sdk/script/management/account-manager.ts

```diff
@@ -73,7 +74,7 @@ export class AccountManager {
                 req = req.send({ identity: "accessKey" })
```

Comment:
I noticed that the 'if' in the collapsed section above still has 'Login the old way' as a case. I think we can just remove it, seeing as all of the keys will be migrated?